services/ui/src/pages/projects.js

```diff
@@ -0,0 +1,41 @@
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
+import ProjectOld from '../components/ProjectOld';
+
+const query = gql`
+{
+  allProjects {
+    id
+    name
+    branches
+    pullrequests
+    created
+    gitUrl
+    productionEnvironment
+    environments {
+      id
+      name
+      environmentType
+      deployType
+      created
+      updated
+    }
+  }
+}
+`;
+
+export default () => <>
+  <h1>Projects</h1>
+  <Query query={query}>
+    {({ loading, error, data }) => {
+      if (loading) return <p>Loading...</p>;
+      if (error) return <p>Error :(</p>;
+
+      const projects = data.allProjects.map(project => <ProjectOld key={project.id} project={project} />);
+
+      return <div>
+        {projects}
+      </div>
+    }}
+  </Query>
+</>
```

Comment:
This whole page is replaced by stuff in `pages/index.js` and can be deleted. `../components/ProjectOld` can also go away, it'll be in git if it's ever useful in the future